storage.bs

```diff
@@ -229,6 +229,17 @@ settings object</a> <var>environment</var>, run these steps:
  <li><p>Return <var>key</var>.
 </ol>
 
+<p>To determine whether a <a>storage key</a> <var>A</var>
+<dfn export for="storage key" id=concept-storage-key-equals lt=equal>equals</dfn> <var>B</var>, run
+these steps:
+
+<ol>
+ <li><p>If <var>A</var>'s <a for="storage key">origin</a> is not <a lt="same origin">same-origin</a>
```

Comment:
"same origin" sans hyphen is what we typically use.

Reply:
I modeled this after https://github.com/whatwg/url/blob/main/url.bs#L2730-L2731, but it looks like "same origin with" is indeed much more commonly used in places like https://html.spec.whatwg.org/multipage/origin.html. Will change!